### PR TITLE
Fix secrets in Crowdin actions

### PR DESCRIPTION
**Pull request by McGiverGim**
The Crowdin upload action, is working when executed directly, by manual request, but not when executed as part of the nightly workflow. This is because the SECRETS environment variables are not available to `workflow_call`.

This PR fixes it adding manually the secrets needed.

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Pull request merged**
